Frends.MicrosoftDynamics365.Request/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [1.1.0] - 2024-07-08
```

Comment:
Last thing that I forgot to mention - this version has to be also updated in `Frends.MicrosoftDynamics365.Request.csproj`